### PR TITLE
Adds reusable component for company logos

### PR DESCRIPTION
**Pull request by francoislehoux-okta**
## Description:
- **What's changed?** Adds reusable component for company logos, shared across all 4 pages displaying them.

### Resolves:
* [OKTA-360425](https://oktainc.atlassian.net/browse/OKTA-360425)

### Screenshots:
Screen Shot 2021-01-14 at 2 08 27 PM
Screen Shot 2021-01-14 at 2 08 33 PM
Screen Shot 2021-01-14 at 2 08 44 PM
Screen Shot 2021-01-14 at 2 08 54 PM

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/index.js [hunk 1]
```diff
@@ -14,9 +14,10 @@ module.exports = (options, ctx) => {
       $page.fullHeaders = resolveHeaders($page);
 
       // Set to true to enable the redesign in development
-      const redesign = process.env.REDESIGN || false;
+      // const redesign = process.env.REDESIGN || false;
       // Respect the redesign flag unless this is a production build
-      $page.redesign = !ctx.isProd && redesign;
+      // $page.redesign = !ctx.isProd && redesign;
+      $page.redesign = true
```
Comment (francoislehoux-okta) [inline packages/@okta/vuepress-theme-prose/index.js:20]: Note

**Comment by jakubvul-okta** [comment]
👌 

No impact on dev doc

File: packages/@okta/vuepress-theme-prose/global-components/SignUp.vue [hunk 2]
```diff
@@ -212,10 +212,7 @@
     <div class="signup--description">
       <Content slot-key="signup-description" />
       <div class="logo-wrapper">
-        <h4>Trusted by</h4>
-        <div class="partners-logo">
-          <img src="/img/authorization/proof-by.png" alt="" />
-        </div>
+        <CompanyLogos withHeading small v-bind:centered="false" />
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/global-components/SignUp.vue:215]: I think you only need the `v-bind:` for props if they are dynamic

**Review by shawyu-okta — APPROVED** [review]

File: packages/@okta/vuepress-theme-prose/global-components/SignUp.vue (hunk 2 above)
Comment (francoislehoux-okta) [reply packages/@okta/vuepress-theme-prose/global-components/SignUp.vue:215]: Without v-bind, it gets treated as a string. [Doc](https://vuejs.org/v2/guide/components-props.html#Passing-a-Boolean)

Comment (shawyu-okta) [reply packages/@okta/vuepress-theme-prose/global-components/SignUp.vue:215]: Wow, love Vue

**Pull request merged**